### PR TITLE
Bump dep warn from silicon to phosphorous for nxos module

### LINKED ISSUE #60669: [TEST FAILURE] Windows test failures
Please paste the link from the specific build where the failure first started:
https://jenkins.saltproject.io/job/Full-PR/job/full-pr-windows-2019-x64/job/PR-1713/3/#showFailuresLink

```
tests.unit.modules.test_nxos.NxosTestCase.test_add_config
tests.unit.modules.test_nxos.NxosTestCase.test_cmd_any_function
tests.unit.modules.test_nxos.NxosTestCase.test_cmd_function_absent
tests.unit.modules.test_nxos.NxosTestCase.test_show_nxapi
tests.unit.modules.test_nxos.NxosTestCase.test_show_nxapi_structured
tests.unit.modules.test_nxos.NxosTestCase.test_show_raw_text_invalid
tests.unit.modules.test_nxos.NxosTestCase.test_show_raw_text_true
tests.unit.modules.test_nxos.NxosTestCase.test_show_raw_text_true_multiple_commands
tests.unit.modules.test_nxos.NxosTestCase.test_system_info
```

### PR DESCRIPTION
**Pull request by xeacott**
### What does this PR do?
Remove deprecation warnings now that Silicon is here

```
RuntimeError: The warning triggered on filename 'C:\Users\Administrator\AppData\Local\Temp\kitchen\testing\salt\modules\nxos.py', line number 479, is supposed to be shown until version 3004 is released. Current version is now 3004. Please remove the warning.
```

### What issues does this PR fix or reference?
Fixes: https://github.com/saltstack/salt/issues/60669

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

File: salt/modules/nxos.py [hunk 1]
```diff
@@ -219,7 +218,6 @@ def cmd(command, *args, **kwargs):
         salt '*' nxos.cmd show_run
         salt '*' nxos.cmd check_password username=admin password='$5$lkjsdfoi$blahblahblah' encrypted=True
     """
-    warn_until("Silicon", "'nxos.cmd COMMAND' is deprecated in favor of 'nxos.COMMAND'")
```
Comment (s0undt3ch) [inline salt/modules/nxos.py:222]: These need a change log entry....

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: salt/modules/nxos.py [hunk 2]
```diff
@@ -382,11 +380,6 @@ def show(commands, raw_text=True, **kwargs):
         salt '*' nxos.show 'show bgp sessions ; show processes' raw_text=False
         salt 'regular-minion' nxos.show 'show interfaces' host=sw01.example.com username=test password=test
     """
-    warn_until(
```
Comment (Ch3LL) [inline salt/modules/nxos.py:385]: Why are we removing the deprecation warnings without removing the code that was scheduled to be removed?

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: salt/modules/nxos.py (hunk 2 above)
Comment (xeacott) [reply salt/modules/nxos.py:385]: Having a conversation with @dmurphy18 about what to do with this..

Comment (xeacott) [reply salt/modules/nxos.py:385]: I'll have to work with David on this one... but the tests will need to be updated and this his territory of stuff :) so if we can reasonably get that done, we'll try. if not, we'll have to punt the deprecation warning to the next release

**Comment by xeacott** [comment]
@Ch3LL - @dmurphy18 and I decided it would be best to bump the deprecation warning to phosphorus

**Review by dmurphy18 — DISMISSED** [review]
Need tests to fully test the new nxos.COMMAND, so agree with punting this to Phosphorus, esp. since Cisco admins are slow to update (past experience).
Also would help to run the tests against an actual Cisco NX-OS router

File: changelog/60669.fixed [hunk 3]
```diff
@@ -0,0 +1 @@
+change dep warn from Si to P
```
Comment (Ch3LL) [inline changelog/60669.fixed:1]:
```suggestion
change dep warn from Silicon to Phosphorus for the cmd,show,system_info and add_config functions in the nxos module.
```

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by s0undt3ch — APPROVED** [review]
Looks good, however, why bump to a future release and not remove the deprecated code?

**Comment by s0undt3ch** [comment]
> Looks good, however, why bump to a future release and not remove the deprecated code?

Nevermind, read comments.

**Review by Ch3LL — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Pull request merged**